src/components/orders/OrderDetails/index.tsx

```diff
@@ -91,11 +99,11 @@ export function OrderDetails(props: Props): JSX.Element {
           </tr>
           <tr>
             <td>Sell amount</td>
-            <td>{`${formatSmart(sellAmount, sellToken.decimals)} ${sellToken.symbol}`}</td>
+            <td>{`${formatSmart(sellAmount.toString(), sellToken.decimals)} ${sellToken.symbol}`}</td>
```

Comment:
ubernit. not important cause it's always hard to make changes to dex-js, and this one is too little. But if we touch it any time soon, maybe we want BigInt support (to save us the toString every time we format ---> no deal at all)

Reply:
Agreed. https://github.com/gnosis/gp-ui/issues/179